packages/signoz/readme.md

```
@@ -0,0 +1,140 @@
+
+- **Traces**: `http://localhost:4318/v1/traces`
+- **Metrics**: `http://localhost:4318/v1/metrics`
+- **Logs**: `http://localhost:4318/v1/logs`
+
+## 3. Accessing SigNoz Dashboard and Trace Explorer
+
+### Dashboard Access
+
+Once SigNoz is running, access the dashboard at:
+
+**URL**: http://localhost:3301
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)